handler.go

```diff
@@ -103,6 +105,30 @@ func newReverseProxy(host string) *httputil.ReverseProxy {
 			r.Out.Host = r.In.Host
 			AdditionalProxyRewrite(r)
 		},
+		Transport: &retryTransport{
```

Comment:
I would prefer to avoid this. Retries at multiple levels (client retry, proxy retry) make things difficult to debug. It will also likely mask problems that should otherwise be solved.

Reply:
done